### PR TITLE
Depthmap toolkit refactor

### PR DESCRIPTION
**Pull request by lvonasek**
# Description

This PR merges convertpcd2depth and pcd2depth together into one script and removes duplicity of intrinsics in depthmap class (as intrinsics object and fx, fy, cx, cy).

### CODE REVIEW

File: cgmml/common/depthmap_toolkit/convertpcd2depth.py [hunk 1]
```diff
@@ -3,16 +3,98 @@
```
Comment (markus-hinsche) [inline cgmml/common/depthmap_toolkit/convertpcd2depth.py:3]: Is the filename still suited or does this code actually do more than just convert pcd to depth?
mabye `conversion_between_pcd_and_depth.py`

File: cgmml/common/depthmap_toolkit/convertpcd2depth.py [hunk 2]
```diff
@@ -3,16 +3,98 @@
...
+            line = str(f.readline())
+            if not line:
+                break
+            else:
+                values = parse_numbers(line)
+                data.append(values)
+    return data
+
+
+def pcd2depth(calibration, pcd_fpath: str, width: int, height: int) -> np.ndarray:
```
Comment (markus-hinsche) [inline cgmml/common/depthmap_toolkit/convertpcd2depth.py:64]: `calibration` doesn't have a type annotation

File: cgmml/common/depthmap_toolkit/depthmap.py [hunk 3]
```diff
@@ -126,7 +126,7 @@ def __init__(
             self.header = header
 
         self.rgb_fpath = rgb_fpath
-        self.rgb_array = rgb_array  # shape (width, height, 3)  # (180, 240, 3)
+        self.rgb_array = rgb_array  # shape (width, height, 3)  # (240, 180, 3)
```
Comment (markus-hinsche) [inline cgmml/common/depthmap_toolkit/depthmap.py:129]: Good catch!

File: cgmml/common/depthmap_toolkit/convertpcd2depth.py (hunk 1 above)
Comment (lvonasek) [reply cgmml/common/depthmap_toolkit/convertpcd2depth.py:3]: it just converts pcd2depth and it is quite deprecated as we do not use PCDs anymore.

**Review by markus-hinsche — APPROVED** [review]
LGTM

**Pull request merged**